tests/test_engines.py

```diff
@@ -12,10 +12,9 @@
     current_dir = os.path.abspath(os.path.dirname(__file__))
     config.cassette_library_dir = os.path.join(current_dir, 'cassettes')
 
-
 class TestDefaultEngine:
     @patch('rasp.base.DefaultEngine._session')
-    def setup(self, req_mock):
+    def setUp(self, req_mock):
```

Comment:
My bad I'll switch it back to setup - I saw nose2 in requirements and thought it was a typo